app/models/publication-flow.js

```diff
@@ -52,6 +47,11 @@ export default class PublicationFlow extends Model {
     return `${this.publicationNumber}`;
   }
 
+  @computed('urgencyLevel')
```

Comment:
In een native class is de `@computed` niet noodzakelijk bij een `get` function. Je hebt dit vermoedelijk overgenomen van `get publicationNumberToDisplay()` maar ook daar is ze overbodig.